merged.nyc.config.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+/******
+ * This file is the configuration for the final merged report.
+ */
+module.exports = {
+	'report-dir': 'coverage/merged',
+	'temp-dir': 'coverage/merged',
+	reporter: ['html', 'json', 'lcov', 'text'],
+	'check-coverage': true,
+	branches: 80,
+	lines: 80,
+	functions: 80,
+	statements: 80,
+	include: ['src/'],
+	exclude: [
+		'cypress/**/*.js',
+		'jest-test-setup.js',
```

Comment:
Can you remove lines 18-20. Let's stick to wildcards here, and then place `/* istanbul ignore file */` at the very top of each file listed below.

src/serviceWorker.js
src/setupTests.js
src/index.js
src/polyfills/index.js
jest-test-setup.js